sdv/metadata/single_table.py

```diff
@@ -477,6 +479,60 @@ def validate(self):
                 + '\n'.join([str(e) for e in errors])
             )
 
+    def visualize(self, show_table_details='full', output_filepath=None):
+        """Create a visualization of the multi-table dataset.
+
+        Args:
+            show_table_details (str):
+                If 'full', show the column names, primary, alternate and sequence keys are all
```

Comment:
remove `show` after `If 'full'`. Also, review the docstrings, they mention table_names when I don't think we have them for single_table.

Also, shouldn't None be an option?

Reply:
I did not add it because it didn't make sense (empty box ?) ... In multi table I see a point of `None` since you can see the diagram.